### PR TITLE
fix(home): socials button overlaps patches button

### PR DESCRIPTION
**Pull request by 404-Program-not-found**

### CODE REVIEW

**Comment by 404-Program-not-found** [comment]
hmm git messed up at some point

**Comment by xafn** [comment]
This can really be split into 3 commits, adding pnpm yaml, style for spacing, and fixing the home alignment.

**Comment by 404-Program-not-found** [comment]
> This can really be split into 3 commits, adding pnpm yaml, style for spacing, and fixing the home alignment.

yeah true, git is a pain to split though. Will see what I can do

**Comment by oSumAtrIX** [comment]
You can `git reset --soft HEAD~X` and then stage and commit them separately.

File: src/app.scss [hunk 1]
```diff
@@ -10,15 +10,20 @@
...
 	overflow-y: scroll;
 }
 
 body {
 	margin: 0;
-	padding: 0;
+	min-height: 100dvh;
+	height: 100%;
+	padding-top: 70px;
+	position: relative;
```
Comment (Ushie) [inline src/app.scss:26]: what's the purpose for this? why is this in body

File: src/app.scss [hunk 2]
```diff
@@ -10,15 +10,20 @@
 
 html {
 	margin: 0;
-	padding: 0;
+	overflow-y: auto;
+	overflow-x: hidden;
+	height: max-content;
```
Comment (Ushie) [inline src/app.scss:15]: what's the purpose of this being in html?

File: src/lib/components/Wave.svelte [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 <style>
 	svg {
-		position: fixed;
+		position: absolute;
```
Comment (Ushie) [inline src/lib/components/Wave.svelte:11]: what did this change?

File: src/routes/+page.svelte [hunk 4]
```diff
@@ -40,10 +40,11 @@
 	.wrap {
 		margin-inline: auto;
 		width: min(87%, 100rem);
-		margin-top: 7rem;
 	}
 	.wrappezoid {
-		height: calc(100vh - 120px);
+		height: 100%;
+		padding-top: 2.5rem;
+		min-height: calc(100vh/10*7);
```
Comment (Ushie) [inline src/routes/+page.svelte:47]: what's this needed for?

File: src/lib/components/Wave.svelte (hunk 3 above)
Comment (404-Program-not-found) [reply src/lib/components/Wave.svelte:11]: this fixes an issue where the wave may cause contrast issues when viewed in landscape on certain devices. For instance: 
image

File: src/app.scss (hunk 2 above)
Comment (404-Program-not-found) [reply src/app.scss:15]: This at one point did solve an issue, however it appears to be no longer necessary and can be safely deleted

File: src/app.scss (hunk 1 above)
Comment (404-Program-not-found) [reply src/app.scss:26]: padding is needed for the navbar.
The rest is needed to fix the issue illustrated below:
image

File: src/routes/+page.svelte (hunk 4 above)
Comment (404-Program-not-found) [reply src/routes/+page.svelte:47]: New calculation makes the home page look more visually balanced regardless of screen size.

Original:
image

New:
image

Comment (Ushie) [reply src/routes/+page.svelte:47]: It seems to move the preview image a bit too high on my device, it's almost touching the navbar

Before:
image

After:
image

File: src/app.scss (hunk 1 above)
Comment (Ushie) [reply src/app.scss:26]: Hm okay, I'm not too good with changes like that so I'll ask afn to review this part

Comment (Ushie) [reply src/app.scss:26]: Wouldn't it be possible to be done in the +page.svelte directly? instead of app.scss

File: src/routes/+page.svelte (hunk 4 above)
Comment (oSumAtrIX) [reply src/routes/+page.svelte:47]: @404-Program-not-found Is this PR still active?

Comment (404-Program-not-found) [reply src/routes/+page.svelte:47]: > @404-Program-not-found Is this PR still active?

yup, I'll look into that. Sorry, life just got a bit busy recently

**Comment by Ushie** [comment]
Hey,

Sorry for the delay in reviewing your PR. I appreciate your time and effort, so I'll go ahead and close this not to waste any more of your time. Thank you for your contribution!

**Pull request closed without merging**